app/Controllers/feedController.php

```diff
@@ -547,6 +547,7 @@ public function actualizeAction() {
 		$url = Minz_Request::param('url');
 		$force = Minz_Request::param('force');
 		$maxFeeds = (int)Minz_Request::param('maxFeeds');
+		$noredirect = (bool)Minz_Request::param('noredirect') ==1;
```

Comment:
```suggestion
		$noredirect = Minz_Request::param('noredirect') == 1;
```

Reply:
Maybe using the following is more appropriated:
```php
$noredirect = Minz_Request::paramBoolean('noredirect');
```